### PR TITLE
Task 70 - Home Page Screenshot

### PR DESCRIPTION
**Pull request by NataSelenium**
Hello Kirill!
Please, review my pull request for task #70.
Thank

### CODE REVIEW

File: src/test/java/task70/page/HomePage.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        PageFactory.initElements(driver, this);
+    }
+
+    public boolean isHomePagePresent() {
+        driver.manage().window().fullscreen();
+        new WebDriverWait(driver, Duration.ofSeconds(10)).until(ExpectedConditions.elementToBeClickable(composeLink));
+        return composeLink.isDisplayed();
+    }
+
+    public WebElement getLoginNameLabel(String name)
```
Comment (KirKarpov) [inline src/test/java/task70/page/HomePage.java]: Method name is 'getLoginNameLabel', but you return WebElement. It is better to return String result. It is not common practice to work with WebElement on the test level(in the test)

File: src/test/java/task70/page/HomePage.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        {
+            driver.manage().window().fullscreen();
+        }
+        getLoginNameLabel(log).click();
+        logOutLink.click();
+        driver.manage().window().fullscreen();
+        return new LoginPage(driver);
+    }
+
+    public void takeHomePageScreenShot(WebDriver driver)
```
Comment (KirKarpov) [inline src/test/java/task70/page/HomePage.java]: Take screenshot method shouldn't be a part of HomePage. Please create new class in Utils package and move logic related to take screenshot to this new class

**Review by KirKarpov — CHANGES_REQUESTED** [review]

File: src/test/java/task70/util/TestConfig.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package task70.util;
+
+public class TestConfig {
+    public static final String LOG_NAME = "WhiteDecember22";
+    public static final String PASSWORD = "GreenMonday22&";
+    public static final String MAIL_YANDEX_URL = "https://mail.yandex.com";
+    public static final String SCREEN_SHORT_FILE = "D:\\JavaProjects\\SeleniumJava\\screenshots\\homePageScreenshot.jpg";
```
Comment (KirKarpov) [inline src/test/java/task70/util/TestConfig.java]: 1) The path should be relative not absolute. 
2) The name of file should be dynamically generated

**Review by KirKarpov — CHANGES_REQUESTED** [review]

File: src/test/java/task70/page/HomePage.java (hunk 1 above)
Comment (NataSelenium) [reply src/test/java/task70/page/HomePage.java]: Done

File: src/test/java/task70/page/HomePage.java (hunk 2 above)
Comment (NataSelenium) [reply src/test/java/task70/page/HomePage.java]: Done

File: src/test/java/task70/util/TestConfig.java (hunk 3 above)
Comment (NataSelenium) [reply src/test/java/task70/util/TestConfig.java]: Done

**Review by KirKarpov — APPROVED** [review]

**Pull request merged**